freqtrade/exchange/__init__.py

```diff
@@ -239,6 +239,19 @@ def get_balances() -> dict:
     except ccxt.BaseError as e:
         raise OperationalException(e)
 
+@retrier
+def get_order_book(pair: str, refresh: Optional[bool] = True) -> dict:
+    try:
+        return _API.fetch_order_book(pair)
```

Comment:
I think you would want to check if the selected exchange actually supports this method (probably at startup - if this config is set?).
if it doesn't you may want to refuse startup ... ?

Reply:
Thanks! I agree on putting a check if this is supported by the exchange... This will be included in the next PR